src/css/application.css

```diff
@@ -131,67 +133,87 @@ body {
   background-color: var(--color-red);
 }
 
-.top-bar__logo-container > svg {
-  height: 3em;
+.top-bar__logo-container {
+  margin-right: 0.2em;
+  min-width: 1.6em;
+  transform: scale(1.4);
+  width: 1.6em;
 }
 
 .top-bar__wordmark-container {
   position: relative;
-  margin-left: -0.25em;
+  margin: 0 1em 0 0.5em;
+  min-width: 6.5em;
+  width: 6.5em;
 }
 
 .top-bar__wordmark-container > svg {
-  padding: 0 1em;
-  max-height: 1em;
-  max-width: 100%;
-  width: auto;
+  width: 100%;
+  height: 100%;
 }
 
 .top-bar__snapshot_in-progress {
-  color: var(--color-light-gray);
+  opacity: 0.5;
   cursor: progress;
 }
 
 .top-bar__avatar {
-  height: 2em;
-  border-radius: 50%;
+  height: 1.8em;
   vertical-align: middle;
-  line-height: 1em;
 }
 
 .top-bar__username {
   margin-left: 0.5em;
 }
 
 .top-bar__menu-button {
-  align-self: stretch;
-  border-color: transparent;
-  border-style: solid;
-  border-width: 0 1px;
+  align-self: center;
   box-sizing: border-box;
   cursor: pointer;
-  line-height: 3em;
-  padding: 0 0.5em;
+  height: 2.8em;
+  line-height: 2.8em;
+  padding: 0 1em;
   position: relative;
   user-select: none;
   vertical-align: middle;
+  margin-right: 0.5em;
+  white-space: nowrap;
+}
+
+.top-bar__menu-button:last-child {
+  margin-right: 0;
+}
+
+.top-bar__menu-button:hover:not(.top-bar__menu-button_active) {
+  box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.25);
+}
+
+.top-bar__menu-button_primary {
+  background-color: rgba(0, 0, 0, 0.5);
+  color: #fff;
+}
+
+.top-bar__menu-button_secondary {
+  box-shadow: inset 0 0 0 0.08em rgba(0, 0, 0, 0.1);
 }
 
 .top-bar__menu-button_active {
-  background-color: white;
-  border-color: var(--color-low-contrast-gray);
+  box-shadow: inset 0 0.08em 0.2em 0 rgba(0, 0, 0, 0.25);
+}
+
+.top-bar__menu-button_projects .top-bar__menu {
+  width: 16rem;
 }
 
 .top-bar__hamburger {
-  font-size: 2em;
+  font-size: 1.5em;
 }
 
 .top-bar__menu {
-  border: 1px solid var(--color-low-contrast-gray);
-  box-shadow: 2px 2px 2px var(--color-low-contrast-gray);
-  left: -1px;
+  box-shadow: 0 0 20px rgba(0, 0, 0, 0.25);
+  right: 0;
   position: absolute;
-  top: 100%;
+  top: 3.4em;
```

Comment:
Where does this number come from? Does 100% not work?

Reply:
I wanted the menus to line up with the bar instead of the button itself. It felt a little crowded and out of place to me.

![o5zp8up6h6](https://user-images.githubusercontent.com/3432105/32761865-cce0b1e4-c8c3-11e7-9c99-8d0269e01e6c.gif)